src/emulator/controller.ts

```diff
@@ -310,9 +310,9 @@ export async function startAll(options: any, noUi: boolean = false): Promise<voi
     }
 
     const rulesLocalPath = options.config.get("firestore.rules");
-    const foundRulesFile = rulesLocalPath && fs.existsSync(rulesLocalPath);
+    const rules: string = path.join(options.projectRoot, rulesLocalPath);
```

Comment:
If `rulesLocalPath` is `undefined` this will fail:
```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at Object.join (path.js:1147:7)
```